lib/saml/url_service.rb

```diff
@@ -59,8 +59,15 @@ def url_settings(options)
       saml_settings(options)
     end
 
-    def saml_options
-      Settings.review_instance_slug.blank? ? {} : { RelayState: Settings.review_instance_slug }
+    def saml_options(alt_relay: false)
+      options = if alt_relay
+                  { RelayState: Settings.saml.relay.alternate }
+                elsif Settings.review_instance_slug.blank? == false
+                  { RelayState: Settings.saml.relay.review_instance_slug }
+                else
+                  {}
+                end
+      options
```

Comment:
I wonder if we have an opportunity to clean this code up a bit if the logic could be based more on the current environment. In other words, is it possible to have these based on environment specific configurations entirely so that we don't need all of this branching logic?

Reply:
AFAIK the plan is to continue to only deploy one instance of vets-api that serves both hostnames, in case that changes your thinking.